test/integration/integration_test_fixtures.py

```diff
@@ -45,24 +47,28 @@ class IntegrationTestBase():
   This class will be refactored (https://github.com/envoyproxy/nighthawk/issues/258).
   """
 
-  def __init__(self, ip_version, backend_count=1):
+  def __init__(self, ip_version, server_config, backend_count=1):
     """
     Args:
       ip_version: a single IP mode that this instance will test: IpVersion.IPV4 or IpVersion.IPV6
       backend_count: number of Nighthawk Test Server backends to run, to allow testing MultiTarget mode
     """
     super(IntegrationTestBase, self).__init__()
-    self.test_rundir = os.path.join(os.environ["TEST_SRCDIR"], os.environ["TEST_WORKSPACE"])
-    self.nighthawk_test_server_path = os.path.join(self.test_rundir, "nighthawk_test_server")
-    self.nighthawk_test_config_path = None
-    self.nighthawk_client_path = os.path.join(self.test_rundir, "nighthawk_client")
+    self.confdir = "nighthawk/test/integration/configurations/"
```

Comment:
We are adding some public attributes, can we document them in the class docstring? It will ideally have an "Attributes:" section.

This is partially pre-existing since none of the other attributes are documented there. Up to you if we document all now, or leave a todo for the pre-existing ones. We should still prefer making anything that can be private.